### ISSUE TITLE
ui-tinymce inside angular-ui/bootstrap tab

### ISSUE THREAD

**Issue by darxmac** [angular-ui/ui-tinymce#168]
When using ui-tinymce inside a ui-bootstrap tabset/tab like this

```html
 <tabset>
    <tab heading="Content">
        <textarea class="form-control" rows="3"  ng-model="myModel.content" ui-tinymce></textarea>
    </tab>
</tabset>
```

`myModel` gets overwritten with `{"content":"<p><br data-mce-bogus=\"1\"></p>"}`

Outside the tab, no problem. Also no problem if using a plain bootstrap tab, i also don't have problems with other directives used inside the tabs

Any ideas ?

**Comment by deeg** [comment]
Please post a Plunker or Fiddle so we can look into fixing the issue.

**Comment by deeg** [comment]
This probably had to do with the format defaulting to 'raw'.

I made a change which will be in the next release, to default to 'html' format and allow the user to override in the options object.

I am going to close this ticket when it gets into master, and if you are still having issues after that please re-open a new ticket with a Plunker or Fiddle attached demonstrating

**Comment by deeg** [comment]
What I believe to be your issue is fixed in the v0.0.11 release. Please check if you are still having problems after updating, and if so open a new issue with a Plunker.
